src/mpid/ch4/netmod/ofi/ofi_types.h

```diff
@@ -502,7 +496,6 @@ typedef struct {
 
 typedef struct MPIDI_OFI_win_request {
     MPIR_OBJECT_HEADER;
-    char pad[MPIDI_REQUEST_HDR_SIZE - MPIDI_OFI_OBJECT_HEADER_SIZE];
     struct fi_context context[MPIDI_OFI_CONTEXT_STRUCTS];       /* fixed field, do not move */
```

Comment:
@hzhou What was the reason that you removed the padding?

Reply:
It appears that `MPIDI_OFI_win_request` is simply an OFI internal object that gets passed around and it is not a regular `MPIR_Request` at all. I have refactored for it to have its own object allocation using `MPIR_INTERNAL` kind type; so I believe the padding is no longer needed.

Reply:
In fact, we could simply use `MPL_malloc` and `MPL_free` for `MPIDI_OFI_win_request` objects :)